apps/coscrad-frontend/src/components/resources/factories/get-id.interface.ts

```diff
@@ -0,0 +1,3 @@
+export interface IGetId {
+    getId: () => string;
+}
```

Comment:
What is the index.tsx in factories? Why is it empty. Should we barrel export? Clean up this dir and consider moving things around.

Reply:
done